src/styles/blocks/content.scss

```diff
@@ -0,0 +1,25 @@
+.container {
+  display: grid;
+  justify-content: center;
```

Comment:
redundant property

Reply:
agree. you don't need to justify-content here